cmd/native-relayer/config.yaml

```diff
@@ -0,0 +1,29 @@
+# Relayer Configuration
+
+# Native
+native_rpc: "127.0.0.1:26657"  
+native_grpc: "127.0.0.1:9090"       
+
+# IKA
+ika_rpc: "127.0.0.1:0001"          
+ika_signer_mnemonic: "" 
+ika_native_lc_package: ""                
+ika_native_lc_module: ""               
+ika_native_lc_function: ""  
+ika_gas_acc: ""                            
+ika_gas_budget: ""                         
+
+# Bitcoin
+btc_rpc_host: "your-bitcoin-rpc.com"           
+btc_rpc_user: "your-bitcoin-rpc-user"          
+btc_rpc_pass: "your-bitcoin-rpc-password"      
+btc_confirmation_threshold: 6  
+http_post_mode: true
+disable_tls: false           
```

Comment:
We should add btc client type and network spec(testnet, mainnet, regtest). This is useful because sometimes we want to modify the relayer related to the type of network and client.

Reply:
Did not think about it, thanks. Added it